### PR TITLE
Huongpham1603/sc 32359/core be dtgk change config bnpl with the new flow

### PR DESCRIPTION
**Pull request by HuongPhamCredify**
## description
[Core] BE DTGK change config BNPL with the new flow

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,4 @@
+SIGNING_KEY=MC4CAQAwBQYDK2VwBCIEIPAAAAA0YTOeUxU45Y7YBhVtdbW/BCeMuxoLCmqcZG8O2KlLf
```
Comment (ngo275) [inline .env.example]: Please remove the credentials

File: .gitignore [hunk 2]
```diff
@@ -132,6 +132,7 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+.env.example
```
Comment (ngo275) [inline .gitignore]: Your gitignore isn't working

File: package.json [hunk 3]
```diff
@@ -16,6 +16,8 @@
     "morgan": "^1.10.0"
   },
   "devDependencies": {
+    "axios": "^1.0.0",
```
Comment (ngo275) [inline package.json:17]: This should be dependencies, not `devDependencies`

File: v1.js [hunk 4]
```diff
@@ -120,16 +127,30 @@ module.exports = () => {
      *   }
      * }
      */
-    const inputs = req.body.inputs
+    let inputs = req.body.inputs;
+    inputs = {
+      ...inputs,
+      product: {
+        manufacturer: "temp",
```
Comment (ngo275) [inline v1.js:134]: Is this OK??

Comment (HuongPhamCredify) [reply v1.js:134]: It is ok. It doesn't have a value yet.

File: package.json (hunk 3 above)
Comment (HuongPhamCredify) [reply package.json:17]: I fixed it

File: .gitignore (hunk 2 above)
Comment (HuongPhamCredify) [reply .gitignore]: I fixed it

File: .env.example (hunk 1 above)
Comment (HuongPhamCredify) [reply .env.example]: I fixed it

File: package.json [hunk 5]
```diff
@@ -13,9 +13,11 @@
     "@credify/nodejs": "^1.21.0",
     "body-parser": "^1.20.0",
     "express": "^4.18.1",
-    "morgan": "^1.10.0"
+    "morgan": "^1.10.0",
+    "axios": "^1.0.0"
   },
   "devDependencies": {
+    "dotenv": "^16.0.3",
```
Comment (ngo275) [inline package.json]: dotenv shoudn't be in `dependencies`?

Comment (HuongPhamCredify) [reply package.json]: I fixed it  :D.

Comment (ngo275) [reply package.json]: Did you update yarn.lock? I mean just updating package.json is not enough. Remove node_modules and yarn.lock, and run `yarn` once again.

Comment (HuongPhamCredify) [reply package.json]: It fixed it

**Review by ngo275 — APPROVED** [review]

**Pull request merged**